controllers/vendors.go

```diff
@@ -32,20 +32,28 @@ import (
 	odfv1alpha1 "github.com/red-hat-data-services/odf-operator/api/v1alpha1"
 )
 
+var StorageClusterKind = odfv1alpha1.StorageKind(strings.ToLower(reflect.TypeOf(ocsv1.StorageCluster{}).Name()) +
+	"." + ocsv1.GroupVersion.String())
+var FlashSystemKind = odfv1alpha1.StorageKind(strings.ToLower(reflect.TypeOf(ibmv1alpha1.FlashSystemCluster{}).Name()) +
+	"." + ibmv1alpha1.GroupVersion.String())
+
+var KnownKinds = []odfv1alpha1.StorageKind{
+	StorageClusterKind,
+	FlashSystemKind,
+}
+
 // VendorStorageCluster returns GroupVersionKind
 func VendorStorageCluster() odfv1alpha1.StorageKind {
```

Comment:
I'd rather not. I did this on purpose because having these as global variables does the string generation only once, and no part of them relies on dynamic values at runtime. These functions were left here only because I didn't think it was worth it to completely remove them as it would have touched even more files than this PR already does. Ideally they'd also be `const`s, but go is the way it is.

Reply:
After an offline discussion, agreed that we should clean this up for leave it for another PR.